lib/jsdom/browser/Window.js

```diff
@@ -538,6 +538,7 @@ function Window(options) {
 
   define(this, {
     name: "nodejs",
+    devicePixelRatio: 1,
```

Comment:
Note that I think these should all be getters (own-getters, but still getters). Happy to continue with them in their current state for now though.